lookout/style/format/feature_extractor.py

```diff
@@ -264,6 +267,8 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None) \
 
         vnodes_parsed_number = sum(len(vn) for vn, _, _ in parsed_files)
         self._log.debug("Parsed %d vnodes", vnodes_parsed_number)
+        if self.vnodes_number_callback is not None:
+            self.vnodes_number_callback(vnodes_parsed_number)
```

Comment:
I am going to split `extract_features` into several functions and you will not have to add this callback. Stay tuned.